test/lib/deployLib.js

```diff
@@ -1,38 +1,44 @@
+            await verifyContract(verification.contracts.modules[module].address, verification.contracts.modules[module].args, verification.contracts.modules[module].contractPath);
+        }
     }
-
-    console.log("Verifying swap handlers");
-    for (let handler of Object.keys(verification.contracts.swapHandlers)) {
-        console.log(handler, verification.contracts.swapHandlers[handler].address, verification.contracts.swapHandlers[handler].args, verification.contracts.swapHandlers[handler].contractPath);
-        await verifyContract(verification.contracts.swapHandlers[handler].address, verification.contracts.swapHandlers[handler].args, verification.contracts.swapHandlers[handler].contractPath);
+    
+    if (Object.keys(verification.contracts.swapHandlers).length > 0) {
+        console.log("Verifying swap handlers");
+        for (let handler of Object.keys(verification.contracts.swapHandlers)) {
+            console.log(handler, verification.contracts.swapHandlers[handler].address, verification.contracts.swapHandlers[handler].args, verification.contracts.swapHandlers[handler].contractPath);
+            await verifyContract(verification.contracts.swapHandlers[handler].address, verification.contracts.swapHandlers[handler].args, verification.contracts.swapHandlers[handler].contractPath);
+        }
     }
-
-    console.log("Verifying euler contracts");
-    for (let contract of Object.keys(verification.contracts)) {
-        if (
-            (verification.contracts[contract].address !== null && verification.contracts[contract].address !== undefined) 
-            && 
-            (verification.contracts[contract].args !== null && verification.contracts[contract].args !== undefined)
-        ) {
-            console.log(contract, verification.contracts[contract].address, verification.contracts[contract].args, verification.contracts[contract].contractPath);
-            await verifyContract(verification.contracts[contract].address, verification.contracts[contract].args, verification.contracts[contract].contractPath);
+    
+    if (Object.keys(verification.contracts).length > 0) {
+        console.log("Verifying euler contracts");
+        for (let contract of Object.keys(verification.contracts)) {
+            if (verification.contracts[contract].address && verification.contracts[contract].args) {
+                console.log(contract, verification.contracts[contract].address, verification.contracts[contract].args, verification.contracts[contract].contractPath);
+                await verifyContract(verification.contracts[contract].address, verification.contracts[contract].args, verification.contracts[contract].contractPath);
+            }
         }
     }
+    
 }
 
 async function verifyContract(contractAddress, contractArgs, contractPath = null) {
     try {
-        if (contractPath === null || contractPath === undefined) {
+        if (contractPath) {
```

Comment:
actions do not match conditions. either negate the condition or switch actions

Reply:
thanks for spotting that. All resolved @kasperpawlowski